src/diff.ts

```diff
@@ -10,11 +11,13 @@ import {
   introspectionFromSchema,
   getIntrospectionQuery
 } from 'graphql';
+import { introspectionQuery } from 'graphql/utilities';
 import fs from 'fs';
 import isGlob from 'is-glob';
 import fetch from 'node-fetch';
 import disparity from 'disparity';
 import { fileLoader, mergeTypes } from 'merge-graphql-schemas';
+import sortSchema from 'sort-graphql-schema';
```

Comment:
The `graphql` package already has a utility function included to sort schemas. Please use that one instead of importing `sort-graphql-schema`. (see https://github.com/graphql/graphql-js/blob/master/src/utilities/lexicographicSortSchema.js)